include/zephyr/drivers/firmware/nrf_ironside/ironside_dvfs.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_ZEPHYR_DRIVERS_FIRMWARE_NRF_IRONSIDE_IRONSIDE_DVFS_H_
+#define ZEPHYR_INCLUDE_ZEPHYR_DRIVERS_FIRMWARE_NRF_IRONSIDE_IRONSIDE_DVFS_H_
+
+#include <stdint.h>
+#include <stddef.h>
+#include <errno.h>
+
+enum ironside_dvfs_oppoint {
+	IRONSIDE_DVFS_OPP_HIGH = 0,
+	IRONSIDE_DVFS_OPP_MEDLOW = 1,
+	IRONSIDE_DVFS_OPP_LOW = 2,
+	IRONSIDE_DVFS_OPP_COUNT
+};
+
+/**
+ * @name IRONside DVFS service error codes.
+ * @{
+ */
+
+/** The requested DVFS oppoint is not allowed. */
+#define IRONSIDE_DVFS_ERROR_WRONG_OPPOINT (EINVAL)
+/** Waiting for mutex lock timed out, or hardware is busy. */
+#define IRONSIDE_DVFS_ERROR_BUSY (EBUSY)
+/** There is configuration error in the DVFS service. */
+#define IRONSIDE_DVFS_ERROR_OPPOINT_DATA (ECANCELED)
```

Comment:
Please use dedicated error codes (starting from 1), we'd rather not depend on a specific errno.h definition.